### PR TITLE
[RecoveryServicesBackup] Fix LRO issue

### PR DESCRIPTION
**Pull request by HarveyLink**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by HarveyLink** [comment]
#37904

File: sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Fix LRO for issue: https://github.com/Azure/azure-sdk-for-net/issues/37904
```
Comment (live1206) [inline sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/CHANGELOG.md:10]: Feel like unnecessary to add the GitHub issue in change log, we can just describe the issue resolved.

File: sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/src/autorest.md [hunk 2]
```diff
@@ -476,4 +476,8 @@ directive:
     where: $.definitions.RecoveryPointProperties.properties.expiryTime
     transform: >
       $["format"] = "date-time";
+  - from: bms.json
```
Comment (live1206) [inline sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/src/autorest.md:479]: Add a TODO here to remove this workaround once swagger got fixed.

File: sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/CHANGELOG.md (hunk 1 above)
Comment (HarveyLink) [reply sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/CHANGELOG.md]: Fixed

File: sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/src/autorest.md (hunk 2 above)
Comment (HarveyLink) [reply sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/src/autorest.md:480]: Fixed

**Review by live1206 — APPROVED** [review]
Is it possible to add a scenario test for this?

**Pull request merged**